webextension/privileged/ui/cloudstorage.js

```diff
@@ -23,25 +23,21 @@ this.cloudstorage = class extends ExtensionAPI {
       context.extension.rootURI.resolve("privileged/ui/CloudDownloadsView.jsm")
     );
 
-    function cleanUpPrefs() {
-      // Ensure cloud storage study prefs are cleared
-      ["lastprompt", "interval.prompt", "storage.key", "api.enabled"].forEach(pref => {
-        Services.prefs.clearUserPref(`cloud.services.${pref}`);
-      });
-    }
+    const cloudStorageEventEmitter = new EventEmitter();
```

Comment:
this appears to be unused

Reply:
This is passed to CloudDownloadView.jsm in line 46 where it's used to send telemetry events

Reply:
Okay but this is in a commit titled "install and uninstall flow" but it looks to not actually be related to the install/uninstall flow.
Who reviewed the first version of this code?

Reply:
You are right it's not related to install/uninstall and showing up because I moved this initialization from line 43 to line 26 in an attempt to organize and keep const declarations together. This code has been reviewed earlier as part of PR  https://github.com/mozilla/shield-cloudstorage/pull/28/ (commit Integrate telemetry probes)